pages/api/Schema/index.ts

```diff
@@ -0,0 +1,15 @@
+import { buildFederatedSchema } from '@apollo/federation';
+import ContactFiltersResolvers from './ContactFilters/resolvers';
+import ContactFiltersTypeDefs from './ContactFilters/typeDefs';
+import ExportContactsResolvers from './ExportContacts/resolvers';
+import ExportContactsTypeDefs from './ExportContacts/typeDefs';
+import TaskAnalyticsResolvers from './TaskAnalytics/resolvers';
+import TaskAnalyticsTypeDefs from './TaskAnalytics/typeDefs';
```

Comment:
Also considering creating a ```index.ts``` for each folder that would export the typeDefs.ts, resolvers.ts, and the dataHandler.ts.

Reply:
Yeah I think that would be a good idea... That might clean up the imports here.

Reply:
@OzzieOrca @reldredge71 Thoughts on this now that we no longer need the typeDefs file and we're using named exports? Could help to clean up imports, but then it's also just an extra file.

Reply:
Yeah I think you're good now, the named exports helps.